concepts/05 UI Components/FileUploader/50 Customize FileUploader.md

```diff
@@ -0,0 +1,162 @@
+
+---
+
+##### jQuery
+
+    <!-- tab: index.html -->
+    <div class="file-uploader-block"></div>
+    <div id="file-uploader"></div>
+
+    <!-- tab: index.js -->
+    $(() => {
+        $('#file-uploader').dxFileUploader({
+            dropZone: '.file-uploader-block',
+        })
+    })
+
+    <!-- tab: styles.css -->
+    .file-uploader-block {
+        width: 350px;
+        height: 200px;
+        background-color: #f5f5f5;
+        border-radius: 16px;
+    }
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <div class="file-uploader-block"></div>
+    <dx-file-uploader dropZone=".file-uploader-block"></dx-file-uploader>
+
+    <!-- tab: app.component.css -->
+    .file-uploader-block {
+        width: 350px;
+        height: 200px;
+        background-color: #f5f5f5;
+        border-radius: 16px;
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div class="file-uploader-block"></div>
+        <DxFileUploader drop-zone=".file-uploader-block" />
+    </template>
+
+    <!-- tab: styles.css -->
+    .file-uploader-block {
+        width: 350px;
+        height: 200px;
+        background-color: #f5f5f5;
+        border-radius: 16px;
+    }
+
+##### React
+
+    <!-- tab: App.tsx -->
+    export default function App() {
+        return (
+            <div class="file-uploader-block"></div>
+            <FileUploader dropZone=".file-uploader-block" />
+        )
+    }
+
+    <!-- tab: styles.css -->
+    .file-uploader-block {
+        width: 350px;
+        height: 200px;
+        background-color: #f5f5f5;
+        border-radius: 16px;
+    }
+
+---
+
+This tutorial places FileUploader inside the custom drop zone container:
+
+---
+
+##### jQuery
+
+    <!-- tab: index.html -->
+    <div class="file-uploader-block">
+        <div id="file-uploader"></div>
+    </div>
+
+    <!-- tab: index.js -->
+    $(() => {
+        $('#file-uploader').dxFileUploader({
+            dropZone: '.file-uploader-block',
+        })
+    })
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <div class="file-uploader-block">
+        <dx-file-uploader dropZone=".file-uploader-block"></dx-file-uploader>
+    </div>
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div class="file-uploader-block">
+            <DxFileUploader drop-zone=".file-uploader-block" />
+        </div>
+    </template>
+
+##### React
+
+    <!-- tab: App.tsx -->
+    export default function App() {
+        return (
+            <div class="file-uploader-block">
+                <FileUploader dropZone=".file-uploader-block" />
+            </div>
+        )
+    }
+
+---
+
+To center FileUploader elements inside a custom drop zone, assign the following CSS styles to the `.dx-fileuploader-input-wrapper` selector:
```

Comment:
But now I assume we're still talking about the same custom drop zone. So I'd say "the drop zone". 

```suggestion
To center FileUploader elements inside the drop zone, assign the following CSS styles to the `.dx-fileuploader-input-wrapper` selector:
```